<launch>
<!-- Planner node. -->
  <node name="planner"
        pkg="automatic_navigation"
        type="motionPlanAndSendGoal.py"
        output="screen">

    <param name="frames/fixed" value="$(arg fixed_frame)" />
    <param name="frames/sensor" value="$(arg sensor_frame)" />

    <param name="topics/vis" value="$(arg vis_topic)" />
    <param name="topics/sensor" value="$(arg sensor_topic)" />
    <param name="topics/grid" value="$(arg grid_topic)" />

    <param name="x/num" value="$(arg x_num)" />
    <param name="x/min" value="$(arg x_min)" />
    <param name="x/max" value="$(arg x_max)" />
    <param name="y/num" value="$(arg y_num)" />
    <param name="y/min" value="$(arg y_min)" />
    <param name="y/max" value="$(arg y_max)" />

    <param name="update/occupied" value="$(arg occupied_update)" />
    <param name="update/occupied_threshold" value="$(arg occupied_threshold)" />
    <param name="update/free" value="$(arg free_update)" />
    <param name="update/free_threshold" value="$(arg free_threshold)" />

    <param name="random_downsample" value="$(arg random_downsample)" />
  </node>
</launch>